<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_window</class>
 <widget class="QMainWindow" name="main_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page"/>
      <widget class="QWidget" name="page_2"/>
      <widget class="QWidget" name="page_3"/>
      <widget class="QWidget" name="page_4"/>
      <widget class="QWidget" name="page_5"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFTP">
    <property name="title">
     <string>FTP</string>
    </property>
   </widget>
   <widget class="QMenu" name="menushow">
    <property name="title">
     <string>可视化</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>工具</string>
    </property>
    <addaction name="actionKb"/>
    <addaction name="actionIP"/>
    <addaction name="actiontrans"/>
   </widget>
   <addaction name="menuFTP"/>
   <addaction name="menushow"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionKb">
   <property name="text">
    <string>Kb</string>
   </property>
  </action>
  <action name="actionIP">
   <property name="text">
    <string>IP配置</string>
   </property>
  </action>
  <action name="actiontrans">
   <property name="text">
    <string>格式转化</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
